src/content/docs/release-notes/docs-release-notes/docs-8-2-2024.mdx

```diff
@@ -0,0 +1,109 @@
+* Added non-ingest AWS Privatelink endpoints to [send New Relic data via AWS PrivateLink](/docs/data-apis/custom-data/aws-privatelink).
+* A thorough clean-up of [PHP agent compatibility and requirements](/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements) makes it much easier for you to understand and find the system compatibiliy and requirements information you need to know.
+
+
+### Minor changes
+
+* Updated [infrastructure agent configuration settings](/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#status-server-enabled) to note that if you install the infrastructure agent via guided install, our CLI installer, then `status` is enabled by default.
+* Finished adding all of the status colors in [introduction to New Relic pathpoint](/docs/new-relic-solutions/business-observability/intro-pathpoint/#touchpoint-status) 
+* We've got new infrastructure UI experiences for [AWS](/docs/infrastructure/amazon-integrations/get-started/introduction-aws-integrations/#insights), [Azure](/docs/infrastructure/infrastructure-integrations/get-started/understand-use-data-infrastructure-integrations/#explore), and [GCP](/docs/infrastructure/google-cloud-platform-integrations/get-started/introduction-google-cloud-platform-integrations/#insights) and updated docs to match.
+* Updated compatibility for [.NET core](/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core) and [.NET Framework](/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework).
+* In [Microsoft SQL Server](/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration/#enable-microsoft-sql-server), it turns out that `VIEW ANY DEFINITION` is the correct name for read access permissions.
+* When you [export a chart as CSV](/docs/query-your-data/explore-query-data/dashboards/dashboards-charts-import-export-data/#export-charts-csv), if you have comments in your NRQL query, you may see unexpected results in the CSV.
+* If your synthetic monitoring scripts are running in an environment with write protected directories and you need to [import and store Node.js modules](/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors/#importing), prepend these paths to appropriate the filenames.
+* Custom event names can include forward slashes, so we've updated [data requirements and limits for custom event data](/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#general) to reflect that.
+* Added the `NEW_RELIC_LAMBDA_HANDLER` environment variable to [introduction to AWS Lambda monitoring](/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/introduction-lambda/#environment). This is used by our handler wrapper to find your function's actual handler.
```

Comment:
Again, either Intro should be capitalized or it should be "the intro...doc", IMO